components/ca-treeview.css

```diff
@@ -0,0 +1,34 @@
+/* GENERIC WEB-COMPONENT OVERRIDES */
+ca-treeview { display: inline-block; vertical-align: top; border-right: none !important; background-color: #E5E8EC; margin: 0; padding: 0.5em; position: absolute; box-sizing: border-box; width: 200px; }
```

Comment:
Yep, my view as well. But this is a copy pasta. I'm less fussed about the CSS initially.